src/pages/index.js

```diff
@@ -51,6 +44,88 @@ let cardFormData = {
   image: "",
 };
 
+// ! INSTANTIATE CLASSES
+
+// Confirm card delete form
+const deleteCardPopup = new PopupDeleteCard({
+  popupSelector: "#confirm-delete-modal",
+});
+
+deleteCardPopup.setDeleteConfirmCallback(handleConfirmDelete);
+
+// New profile image form / Avatar
+const profileImagePopup = new PopupWithForm(
+  "#profile-image-modal",
+  handleProfileImageSubmit
+);
+
+// Edit profile form
+const profileEditPopup = new PopupWithForm(
+  "#profile-edit-modal",
+  handleProfileEditSubmit
+);
+
+// Add card form
+const addCardPopup = new PopupWithForm(
+  "#add-card-modal",
+  handleAddCardSubmit,
+  // new handler to allow for saving of data on modal close
+  updateCardFormData
+);
+
+// Full image popup
+const imagePreviewPopup = new PopupWithImage("#card-image-modal");
+
+// API
+const api = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  headers: {
+    authorization: "298fcaaa-e173-4e81-9326-ea95e21a4625",
+    "Content-Type": "application/json",
+  },
+});
+
+// User Info
+const user = new UserInfo({
+  title: ".profile__title",
+  subtitle: ".profile__subtitle",
+  avatar: ".profile__image",
+});
+
+// Creates card section
+const cardSection = new Section(
+  {
+    items: [],
+    renderer: (item) => {
+      const card = createCard(item);
+      cardSection.addItem(card.getCardElement());
```

Comment:
`.getCardElement()` should be inside `createCard` not to duplicate it in 2 different places


```js
    renderer: (item) => {
      const card = createCard(item);
      cardSection.addItem(card);
    },
```